src/client.js

```diff
@@ -840,9 +841,9 @@ MatrixClient.prototype.getStoredDevicesForUser = async function(userId) {
  * @param {string} userId the user to list keys for.
  * @param {string} deviceId unique identifier for the device
  *
- * @return {Promise<?module:crypto/deviceinfo>} device or null
+ * @return {module:crypto/deviceinfo} device or null
  */
-MatrixClient.prototype.getStoredDevice = async function(userId, deviceId) {
```

Comment:
There are various places that `await` these that no longer need to do so in JS SDK... Did you want to clean those up?

Reply:
ah yes - think I've got them all now